config/crd/bases/configuration.konghq.com_kongingresses.yaml

```diff
@@ -160,20 +170,16 @@ spec:
                           codes to mark targets unhealthy.
                         properties:
                           http_failures:
-                            minimum: 0
                             type: integer
                           http_statuses:
                             items:
                               type: integer
                             type: array
                           interval:
-                            minimum: 0
                             type: integer
                           tcp_failures:
-                            minimum: 0
                             type: integer
-                          timeout:
-                            minimum: 0
+                          timeouts:
```

Comment:
So it looks like this was previously a typo, and broken as the `go-kong` JSON attrs show the plural:

```
`json:"timeouts,omitempty" yaml:"timeouts,omitempty"`
```

Seems like we should simply note this in the changelog as fixed so that if anyone tried to update to an earlier version and had a failure on this option they will be able to easily find where it was fixed.

Reply:
41271fd7